Test/Test/ConsoleReader.h

```diff
@@ -1,30 +1,13 @@
 #pragma once
-#include <iostream>
-#include <conio.h>
+
+const char esc = 27;
+const char enter = 13;
+const char backspace = 8;
 
 class ConsoleReader {
-	public:
+	private:
 		std::string buffer = "";
-		int readConsole() {
-			char esc = 27;
-			char enter = 13;
-			char backspace = 8;
-			char c;
-			do {
-				c = _getch();
-				if (c != enter && c != esc) {
-					if (c != backspace)
-						buffer += c;
-					else
-						buffer.pop_back();
-					_putch(c);
-				}
-				// can getchar even get esc?
-				else
-					break;
-			} while (true);
-			if (c == esc)
-				buffer = "";
-			return buffer.length();
-		};
+	public:
+		std::string getData();
```

Comment:
this method can be const, since it doesn't change internal state of ConsoleReader instance

Reply:
Thank you, fixed